Sources/http.swift

```diff
@@ -20,16 +20,17 @@ public class HTTP {
 }
 
   init(port: UInt16) {
-    serverSocket = socket(AF_INET, Int32(SOCK_STREAM.rawValue), 0)
+    serverSocket = socket(AF_INET, Int32(SOCK_STREAM), 0)
```

Comment:
Have you tested this on Linux? Because when I tried to build on Linux, this line will caused some errors